fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -14,20 +15,17 @@
  * An implementation of Physical table that is backed by a single fact table.
  */
 public class ConcretePhysicalTable extends BasePhysicalTable {
-
     /**
      * Create a concrete physical table.
      * The availability on this table is initialized to empty intervals.
      *
-     * @param name  Fili name of the physical table
-     * @param factTableName  Name of the associated table in Druid
+     * @param name  Name of the physical table as TableName
      * @param timeGrain  time grain of the table
      * @param columns  The columns for this table
      * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
      */
     public ConcretePhysicalTable(
-            @NotNull String name,
-            @NotNull String factTableName,
+            @NotNull TableName name,
```

Comment:
actually, it's not encouraged in java to use type names as part of the variable name unless it disambiguates.  In this case, the name is a pretty good description of what this is.